src/config/dynamic-theme-fixes.config

```diff
@@ -5141,6 +5141,9 @@ IGNORE IMAGE ANALYSIS
 
 reddit.com
 
+INVERT
+svg._1bWuGs_1sq4Pqy099x_yy-
```

Comment:
The class is actuall dynamiclly generated and therefor not a reliable fix.
I think something like
```CSS
IGNORE INLINE STYLE
header a svg > g
header a svg > g > path
```
Should fix the issue.